client/state/current-user/gravatar-status/selectors.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import has from 'lodash/has';
+
+export function isCurrentUserUploadingGravatar( state ) {
```

Comment:
Selectors will probably be used the most by other folks, so JSDoc is usually handy here.